server/schedulers/balance_region.go

```diff
@@ -214,20 +218,20 @@ func (s *balanceRegionScheduler) Schedule(cluster opt.Cluster) []*operator.Opera
 }
 
 // transferPeer selects the best store to create a new peer to replace the old peer.
-func (s *balanceRegionScheduler) transferPeer(plan *balancePlan) *operator.Operator {
+func (s *balanceRegionScheduler) transferPeer(plan *balancePlan, dstStores []*core.StoreInfo) *operator.Operator {
+	excludeTargets := plan.region.GetStoreIds()
+	for _, v := range plan.excludeTarget {
+		excludeTargets[v.GetID()] = struct{}{}
+	}
 	filters := []filter.Filter{
-		filter.NewExcludedFilter(s.GetName(), nil, plan.region.GetStoreIds()),
+		filter.NewExcludedFilter(s.GetName(), nil, excludeTargets),
 		filter.NewPlacementSafeguard(s.GetName(), plan.cluster, plan.region, plan.source),
-		filter.NewRegionScoreFilter(s.GetName(), plan.source, plan.cluster.GetOpts()),
```

Comment:
Where is NewRegionScoreFilter now?

Reply:
The score should sort once in source selection, and the targets's score must  less than the source by index.

Reply:
Please remove NewRegionScoreFilter then.